src/Microsoft.AspNet.Mvc.Localization/project.json

```diff
@@ -15,17 +15,5 @@
   "frameworks": {
     "dnx451": {},
     "dnxcore50": {}
-  },
-  "exclude": [
-    "wwwroot",
-    "node_modules",
-    "bower_components"
-  ],
```

Comment:
Do not remove `exclude` from any project. That pattern group affects many other `dnu` commands and helps us avoid `PathTooLongException`s.